operator/duties/scheduler.go

```diff
@@ -300,7 +300,11 @@ func (s *Scheduler) HandleHeadEvent(logger *zap.Logger) func(event *eth2apiv1.Ev
 		var zeroRoot phase0.Root
 
 		data := event.Data.(*eth2apiv1.HeadEvent)
-		if data.Slot != s.network.Beacon.EstimatedCurrentSlot() {
+
+		// we are interested only in events for the current slot, but to account for wall-clock
+		// differences the next slot after "what we think is the current one" can also be valid
+		if data.Slot != s.network.Beacon.EstimatedCurrentSlot() &&
+			data.Slot != s.network.Beacon.EstimatedCurrentSlot()+1 {
 			return
 		}
```

Comment:
I think this also needs a slight adjustment (like I suggest in this PR) because "current slot" is based on our own "local current time" which might slightly lag behind "true current time" (there is no such thing as true time in distributed systems really, but what I mean is somebody sends us an event slightly earlier than we expect it and considers this OK to do because "his clock is running early").

Not sure how likely it is to happen in practice though - still better safe than sorry.